### ISSUE TITLE
Blockchain not syncing

### ISSUE THREAD

**Issue by Quaternioneer** [gcash/android-neutrino#43]
I have been tapping the Blockchain rectangle in Settings throughout the day, but it stop at "Height: 661000" and "Scanned Through: 0". There hasn't been any commits in a year. Is this project on hiatus or abandoned?

**Comment by Sounds-Like** [comment]
I have the same issue. Stuck at 661000.
Is it possible to get the seed form the wallet to use another wallet to get out the funds?

**Comment by Quaternioneer** [comment]
Yes, go to settings > Backup wallet > enter this seed into another wallet which uses BIP 39.

**Comment by Sounds-Like** [comment]
It is not possible. I have no such option "Backup wallet". I only have recover wallet.
